DEVELOPER_GUIDE.md

```diff
@@ -115,12 +242,12 @@ If you want to build a specific platform, pass the platform flag after `yarn bui
 yarn build-platform --darwin
 ```
 
-You could pass one or multiple flags. If you don't pass any flag, `yarn build-platform` will build an artifact based on your local environment. 
+You could pass one or multiple flags. If you don't pass any flag, `yarn build-platform` will build an artifact based on your local environment.
 
 Currently, the supported flags for this script are:
 * `darwin` (builds Darwin x64)
 * `linux` (builds Linux x64)
-* `linux-arm` (builds Linux ARM64).  
+* `linux-arm` (builds Linux ARM64).
```

Comment:
should we include windows now too?

Reply:
Is that a supported flag now? Sounds like we missed a doc update. Will add, but we may want to re-visit in a separate issue/PR.